src/xdocs/config_javadoc.xml

```diff
@@ -3608,6 +3616,39 @@ public int foobar() {
               <td>6.2</td>
             </tr>
           </table>
+          <br/>
+          <p>
+            As per <a href="https://docs.oracle.com/javase/10/docs/specs/doc-comment-spec.html#inline-tags">
+            Documentation Comment Specification for the Standard Doclet</a> allowed tags inside
+            inline summary javadoc are
+          </p>
+          <table>
+            <tr>
+              <td>name</td>
+              <td>values</td>
+            </tr>
+            <tr>
+              <td>ALLOWED_INLINE_TAGS</td>
+              <td>
+                <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.html#CODE_LITERAL">
+                  CODE_LITERAL</a>,
+                <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.html#LINK_LITERAL">
+                  LINK_LITERAL</a>,
+                <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.html#INPUT_TAG">
+                  INPUT_TAG</a>,
+                <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.html#DOC_ROOT_LITERAL">
+                  DOC_ROOT_LITERAL</a>,
+                <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.html#LINKPLAIN_LITERAL">
+                  LINKPLAIN_LITERAL</a>,
+                <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.html#LITERAL_LITERAL">
+                  LITERAL_LITERAL</a>,
+                <a href="https://checkstyle.org/apidocs/com/puppycrawl/tools/checkstyle/api/JavadocTokenTypes.html#VALUE_LITERAL">
+                  VALUE_LITERAL</a>,
+                @index
```

Comment:
I thought on this and decided that we need to update all above tags to be `@xxxxx` as it is documentation for users and user should not be aware how such tags are represented internally. Lets be close to user and how he see a code, not how chekcstyle/Check see code internally.

Reply:
Done